src/test/resources/jenkins/security/plugins/ldap/casc.yml

```diff
@@ -1,3 +1,5 @@
+configuration-as-code:
+  deprecated: warn
```

Comment:
Purpose?

Reply:
It is now removed and the casc.yml now reflects the current available properties